Challenge1/README.md

```diff
@@ -12,14 +12,22 @@ This bot detects Forta bots deployed by Nethermind.
 ## Alerts
 
 - NEW-BOT-DEPLOYED
+
   - Fired when the `createAgent` function in the Forta Registry contract is called by the Nethermind EOA, i.e. a new Forta Bot is deployed by Nethermind
   - Severity is always set to "low"
   - Type is always set to "info"
   - Metadata: agentId (string)
```

Comment:
Check out how we usually do sub-bullet points for the `metadata`. Also, note our usage of ticks (`).